File: 04-labs/2026-09-07-lab3-crash-workshop/broken-scripts/03_name.py
# -----------------------------------------------------------------
# 03 -- Mean quantity per exported flow
# -----------------------------------------------------------------

import pandas as pd

trade = pd.read_csv("data/raw/comtrade_fr_roundwood_clean.csv")

exports = trade[trade.flowDesc == "Export"]

print("Mean exported quantity:", export.qty.mean(), "m3")

File: 04-labs/2026-09-07-lab3-crash-workshop/broken-scripts/03_name.R
# -----------------------------------------------------------------
# 03 -- Quantite moyenne par flux exporte
# -----------------------------------------------------------------

trade <- read.csv("data/raw/comtrade_fr_roundwood_clean.csv")

exports <- subset(trade, flowDesc == "Export")

cat("Quantite moyenne exportee :", mean(export$qty, na.rm = TRUE), "m3\n")
